tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -15,7 +15,10 @@ class WP_REST_Global_Styles_Controller_Test extends WP_Test_REST_Controller_Test
 	 * @var int
 	 */
 	protected static $admin_id;
-
+	/**
+	 * @var int
+	 */
+	protected static $editor_id;
 	/**
```

Comment:
Nitpick much? Yes.

```suggestion

	/**
	 * @var int
	 */
	protected static $editor_id;

	/**
```

Reply:
I welcome picking all the nits! Thanks 🙇🏻